clear_cell_outputs/README.md

```diff
@@ -0,0 +1,79 @@
+# clear cell outputs
+
+This example shows how to clear all cell outputs at once by clicking on the button.
+
+![Github Actions Status](https://github.com/yash112-lang/extension-examples/blob/master/clear_cell_outputs/Preview.gif)
+
+<!-- A JupyterLab extension for clearing all cells output at once. -->
+
+To use it first we need to import the packages
+```
```

Comment:
We are using a tool call embedme to automatically copy the snippet from the code. So we are sure that the code and the README are synchronized. To use it you need to specify the language type. Then start the block with a comment that provides the file and the line range that should be included like this:

```suggestion
```ts
// src/index.ts#L1-L19

```

Finally you need to run from the top folder:

```
jlpm run embedme
```